Vagrantfile

```diff
@@ -1,4 +1,4 @@
-OS = (ENV['OS'] || "alpine310")
```

Comment:
IMO, revert this and save for a different changeset/PR that takes into account the need for a directory entry or symlink to `scripts/provision/generic/alpine310`

Reply:
I'm going to commit this as-is since we've already done this on all the release branches. I'll put together a PR to fix the provisioning script, which we can backport to the release branches.